src/remediations/controller.fifi.js

```diff
@@ -87,11 +87,17 @@ exports.listPlaybookRuns = errors.async(async function (req, res) {
     const {column, asc} = format.parseSort(req.query.sort);
     const {limit, offset} = req.query;
     let remediation = await queries.getPlaybookRuns(req.params.id, req.user.account_number, req.user.username, column, asc);
+    const rhcRuns = await fifi.getRHCRuns();
```

Comment:
This will retrieve the entire history of playbook runs, even for other remediations, right?